patternfly/patternfly-elements
# ## clean up scattered build files

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

 - remove `elements/*/.editorconfig` files; it is only needed at the root
 - remove `elements/*/.npmignore` files; they all contained nothing but `node_modules` which is [ignored by default](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package).
 - remove `elements/*/.travis.yml` files; it is only needed at the root
 - remove `elements/*/.babelrc` files; it is only needed at the root

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. `npm install` succeeds
2. `npm test` passes
3. `npm start` and poke around, make sure elements work as expected
4. If your editor supports Editor Config, open a source file and make sure the `.editorconfig` conventions are enforced, like tab width.

### Ready-for-merge Checklist

- [ ] Did automated tests pass?
- [ ] Did it get a LGTM *after the last commit*?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
